contrib/kind.sh

```diff
@@ -260,39 +285,60 @@ print_params() {
 
 command_exists() {
   cmd="$1"
-  which ${cmd} >/dev/null 2>&1
+  command -v ${cmd} >/dev/null 2>&1
+}
+
+install_j2_renderer() {
+  # ensure j2 renderer installed
+  pip install wheel --user
+  pip freeze | grep j2cli || pip install j2cli[yaml] --user
+  export PATH=~/.local/bin:$PATH
 }
 
 check_dependencies() {
-  for cmd in pip jq kind ; do
-    if ! command_exists ${cmd} ; then
-  	  echo "Dependency not met: Command not found '${cmd}'"
-  	  exit 1
+  if ! command_exists kind ; then
+    echo "Dependency not met: Command not found 'kind'"
+    exit 1
+  fi
+
+  if ! command_exists jq ; then 
+    echo "Dependency not met: Command not found 'jq'"
+    exit 1
+  fi 
+
+  if ! command_exists j2 ; then 
+    if ! command_exists pip ; then 
+      echo "Dependency not met: 'j2' not installed and cannot install with 'pip'"
+      exit 1
     fi
-  done
+    echo "'j2' not found, installing with 'pip'"
+    install_j2_renderer
+  fi 
 
   if ! command_exists docker && ! command_exists podman; then
   	  echo "Dependency not met: Neither docker nor podman found"
   	  exit 1
   fi
 }
 
-install_j2_renderer() {
-  # ensure j2 renderer installed
-  pip install wheel --user
-  pip freeze | grep j2cli || pip install j2cli[yaml] --user
-  export PATH=~/.local/bin:$PATH
-}
-
 set_default_params() {
   # Set default values
+  # Used for multi cluster setups 
   KIND_CLUSTER_NAME=${KIND_CLUSTER_NAME:-ovn}
+  export KUBECONFIG=${KUBECONFIG:-${HOME}/${KIND_CLUSTER_NAME}.conf}
+  MANIFEST_OUTPUT_DIR=""
+  if [ ${KIND_CLUSTER_NAME} != "ovn" ]; then 
+    MANIFEST_OUTPUT_DIR="../yaml/${KIND_CLUSTER_NAME}"
```

Comment:
Shouldn't you use $DIR to use as an anchor path and then go relative to it? That will allow this script to be called from anywhere?

Reply:
Done